deploy/metrics.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: allocation-api
+  namespace: offender-management-staging
+  labels:
+    k8s-apps: allocation-api
```

Comment:
This label is not required and `k8s-app[s]` is typically used by kubernetes components deployed in the cluster. Although it won't affect kube in any way, can you please rename the name of the label if you plan to keep it?

Reply:
👌